pkg/labels/filter.go

```diff
@@ -34,13 +34,31 @@ type LabelPrefix struct {
 	Source string `json:"source"`
 }
 
+func ParseLabelPrefix(label string) *LabelPrefix {
+	labelPrefix := LabelPrefix{}
+	t := strings.SplitN(label, ":", 2)
+	if len(t) > 1 {
+		labelPrefix.Source = t[0]
```

Comment:
Should we validate the source? What happens if user misspells `cilium` for example?